product_packaging_multi_barcode_stock_menu/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Multiple barcodes menu show product packaging",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Camptomcamp, Odoo Community Association (OCA)",
+    "category": "Product Management",
+    "depends": ["product_multi_barcode_stock_menu"],
```

Comment:
Need to add `product_packaging_multi_barcode` as well here

Reply:
@sebalix dependency was added.